Sources/xcproj/XCScheme.swift

```diff
@@ -251,7 +251,7 @@ final public class XCScheme {
 
     // MARK: - Build Action
 
-    public class SerialAction {
+    public class SerialAction: AutoEquatable {
```

Comment:
I'm having some issues with Sourcery not generating the extensions for the subclasses of `XCScheme.SerialAction`. Do you know @ilyapuchka if the following `for` loop works for nested types?

```
{% for type in types.implementing.AutoEquatable|class|!annotated:"skipEquality" %}
```